From 6a8d25a3969be42b9eab9c386291ba1052538383 Mon Sep 17 00:00:00 2001
From: Oskari Lemmela <oskari@lemmela.net>
Date: Sat, 4 Jul 2020 16:11:23 +0300
Subject: [PATCH] configs: pine64-lts: enable environment in spi flash

enable config options for spi flash device and
environment in spi flash.

Signed-off-by: Oskari Lemmela <oskari@lemmela.net>
---
 configs/pine64-lts_defconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/configs/pine64-lts_defconfig b/configs/pine64-lts_defconfig
index 7b6d5f32c0..c3b34da0d9 100644
--- a/configs/pine64-lts_defconfig
+++ b/configs/pine64-lts_defconfig
@@ -14,3 +14,13 @@ CONFIG_DEFAULT_DEVICE_TREE="sun50i-a64-pine64-lts"
 CONFIG_SUN8I_EMAC=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_CMD_MTD=y
+CONFIG_CMD_MTDPARTS=y
+CONFIG_DM_MTD=y
+CONFIG_ENV_IS_IN_SPI_FLASH=y
+CONFIG_ENV_OFFSET=0xE0000
+CONFIG_ENV_SECT_SIZE=0x20000
+CONFIG_MTD=y
+CONFIG_MTDIDS_DEFAULT="nor0=spi_flash"
+CONFIG_MTDPARTS_DEFAULT="mtdparts=spi_flash:896k(uboot)ro,128k(env),-(firmware)"
+CONFIG_SPI_FLASH_MTD=y
-- 
2.25.1

